backend/src/main/java/com/benine/backend/http/PresetsCreatePresetHandler.java

```diff
@@ -0,0 +1,135 @@
+package com.benine.backend.http;
+
+import com.benine.backend.LogEvent;
+import com.benine.backend.Preset;
+import com.benine.backend.PresetController;
+import com.benine.backend.ServerController;
+import com.benine.backend.camera.Camera;
+import com.benine.backend.camera.CameraConnectionException;
+import com.benine.backend.camera.CameraController;
+import com.benine.backend.camera.Position;
+import com.benine.backend.camera.ipcameracontrol.IPCamera;
+import com.benine.backend.video.StreamController;
+import com.benine.backend.video.StreamNotAvailableException;
+import com.benine.backend.video.StreamReader;
+import org.eclipse.jetty.server.Request;
+
+import java.awt.image.BufferedImage;
+import java.io.File;
+import java.io.IOException;
+import java.sql.SQLException;
+import javax.imageio.ImageIO;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+public class PresetsCreatePresetHandler extends RequestHandler {
+
+  /**
+   * Constructor for a new PresetsCreatePresetHandler, handling the /presets/createpreset request.
+   */
+  public PresetsCreatePresetHandler() {}
+
+  @Override
+  public void handle(String s, Request request, HttpServletRequest req, HttpServletResponse res)
+          throws IOException, ServletException {
+    try {
+      String camID = request.getParameter("camera");
+      if (camID == null) {
+        throw new MalformedURIException("No Camera ID Specified.");
+      }
+
+      CameraController cameraController = ServerController.getInstance().getCameraController();
+      Camera camera = cameraController.getCameraById(Integer.parseInt(camID));
+      System.out.println(ServerController.getInstance().getCameraController().getCameraById(1));
+
+      if (camera instanceof IPCamera) {
+        IPCamera ipcam = (IPCamera) camera;
+        System.out.println(ipcam);
```

Comment:
System.out again? I think it would be better to either remove this or change this to the logger.